docs/release-notes/v1.50-v1.59.md

```diff
@@ -116,6 +334,7 @@ Please review the documentation, as these changes may impact your integrations.
     1. **Tabular Data Endpoint Changes**
 
           * ??? abstract "Introduced changes to Tabular Data"
+
```

Comment:
Could you please remove this newline? It has made the expandable section empty. Or was this intentional?

Reply:
Additionally, please remove all the added newlines added below.